closed/src/java.base/share/classes/java/io/ClassByNameCache.java

```diff
@@ -63,8 +61,11 @@ public ClassByNameCache() {
             loader = loader.getParent();
         }
         setCanonicalSystemLoaderRef(null);
-        AccessController.doPrivileged(
-                new CreateReaperAction(this, staleLoaderRefs)).start();
+
+        Reaper reaper = new Reaper(this, staleLoaderRefs);
+        com.ibm.oti.vm.VM.getVMLangAccess()
+            .createThread(reaper, "ClassByNameCache Reaper", true, false, true, null)
+            .start();
```

Comment:
Continued lines are indented 8 extra spaces (see line 191, and elsewhere in this file).